extensions/wikia/FacebookClient/tests/FacebookClientTest.php

```diff
@@ -1,47 +0,0 @@
-<?
```

Comment:
Moved all these tests into `UserLoginTest.php` and updated the method they call.